benchmarks/cupy/bench_fusion.py

```diff
@@ -0,0 +1,117 @@
+import functools
+import inspect
+
+from benchmarks.numpy.common import Benchmark
+from benchmarks.utils import sync
+from benchmarks.utils.helper import parameterize
+
+import cupy
+
+
+def fuse_shapes(*shape_args, **shapes_dict):
+    def decorator(func):
+        fused_func = cupy.fuse(func)
+        @functools.wraps(func)
+        def wrapper(self, fusion_param):
+            if fusion_param == 'enabled':
+                fused_func(*args)
+            elif fusion_param == 'disabled':
+                func(*args)
+            elif fusion_param == 'compile':
+                raise NotImplementedError
+                # TODO (imanishi)
```

Comment:
Conventionally no spaces between TODO and name.
(This is not a documented rule but culture coming of Google's Style Guide: https://google.github.io/styleguide/pyguide.html#TODO_Comments)

Reply:
Fixed.